runtime/syntax/generator/vim.vim.base

```diff
@@ -1399,8 +1405,8 @@ if !exists("skip_vim_syntax_inits")
  hi def link vimQuoteEscape	vimEscape
  hi def link vimRegister	SpecialChar
  hi def link vimScriptDelim	Comment
```

Comment:
doesn't matter for this PR, but shouldn't `vimScriptDelim` also be linked to `Delimiter`?